addons/web/static/src/views/fields/properties/property_definition_selection.xml

```diff
@@ -6,7 +6,10 @@
             <div class="w-100">
                 <div
                     t-foreach="optionsVisible" t-as="option" t-key="option[0]"
-                    class="o_field_property_selection_option d-flex flex-row align-items-center justify-content-between mb-2 rounded">
+                    class="o_field_property_selection_option d-flex flex-row align-items-center justify-content-between mb-2 rounded"
+                    t-att-option="option[0]">
```

Comment:
Maybe "option-value" or "option-id" and not just "option" ?

Reply:
ok for `option-name`